cmd/server/assets/home.html

```diff
@@ -260,6 +260,18 @@ <h1>Create verification code</h1>
       $uuidConfirm = $('#uuid-confirm');
         $uuid = $('#uuid');
 
+      {{if $hasSMSConfig}}
+      // Initialize pretty phone
+      let phone = document.querySelector('#phone');
+      let iti = window.intlTelInput(phone, {
+        nationalMode: true,
+        {{- if $currentRealm.SMSCountry }}
```

Comment:
...I should totally be making more use of `{{-` for nicer layout

Reply:
I really try to avoid it unless specifically required. I don't particularly care in HTML, but js is... finicky.